src/ngraph/node_input.cpp

```diff
@@ -152,15 +153,35 @@ namespace ngraph
     bool Input<const Node>::operator>=(const Input& other) const { return !(*this < other); }
     std::ostream& operator<<(std::ostream& out, const Input<Node>& input)
     {
-        return input.get_node()->write_description(out, 0)
-               << ".input(" << input.get_index() << "):" << input.get_element_type()
-               << input.get_partial_shape();
+        if (is_type<op::GetOutputElement>(input.get_node()))
+        {
+            return input.get_node()->write_description(out, 0)
+                   << "(" << input.get_source_output().get_node()->get_name() << "["
+                   << input.get_source_output().get_index() << "]):" << input.get_element_type()
+                   << input.get_partial_shape();
+        }
+        else
+        {
+            return input.get_node()->write_description(out, 0)
+                   << ".input(" << input.get_index() << "):" << input.get_element_type()
+                   << input.get_partial_shape();
+        }
     }
 
     std::ostream& operator<<(std::ostream& out, const Input<const Node>& input)
     {
-        return input.get_node()->write_description(out, 0)
-               << ".input(" << input.get_index() << "):" << input.get_element_type()
-               << input.get_partial_shape();
+        if (is_type<op::GetOutputElement>(input.get_node()))
```

Comment:
Why not define `write_description` for `GetOutputElement` ?

Reply:
Rewritten